src/client/version/client.go

```diff
@@ -42,6 +42,15 @@ func PrettyPrintVersion(version *pb.Version) string {
 	return result
 }
 
+// This returns true if Pachyderm is at least at the given version. This allows
+// us to gate backwards-incompatible features on release boundaries.
+//
+// If version.IsAtLeast() is called anywhere in the code with an old version, it
+// can safely be assumed to be "true" and removed
+func IsAtLeast(major, minor int) bool {
```

Comment:
What about revision numbers? Do they not matter in this context?

Reply:
I came up with this function in the context of gating backwards-incompatible features. In that context I don't think revision numbers matter, just because revisions are our most granular releases, so there's no sense in gating a change on a revision vs just merging it.

I'm not opposed to adding a revision argument, though, if you think it could be useful

Reply:
That makes sense